src/cli/cmds/new.ts

```diff
@@ -308,7 +290,7 @@ function yarnAdd(baseDir: string, spec: string): string {
  * @returns The string that includes the install command ("yarn add ...")
  */
 function renderInstallCommand(dir: string, module: string): string {
-  return `yarn add --modules-folder=${dir}/node_modules --silent --no-lockfile --ignore-engines --dev ${module}`;
+  return `npm i --silent -D -f --no-package-lock --prefix=${dir} ${module}`;
```

Comment:
ℹ️  Note the `-f`. If this is not used, then there are almost always going to be peer dependency issues failures. Most external projects have projen as a peer dependency, but since projen has a 0 major version number and moves quickly, it's pretty impossible to satisfy so NPM install will fail. Yarn didn't install peer deps, so it avoided this. In case the external projects has peer deps necessary for synthesis, we should make sure to install them.

Reply:
Hmm, I wonder if the silent flag really helps us at all. Maybe take it out

Also since we don't have to type this command ourselves, might as well spell out the arguments to improve readability. 🙂 
```suggestion
  return `npm install --save-dev -f --no-package-lock --prefix=${dir} ${module}`;
```